integration/spark2/src/main/scala/org/apache/carbondata/store/SparkCarbonStore.scala

```diff
@@ -110,8 +110,16 @@ class SparkCarbonStore extends MetaCachedCarbonStore {
   }
 
   def startSearchMode(): Unit = {
```

Comment:
I think the log can be optimized like below:

```
"Starting search mode master"
```
```
s"Waiting search mode master to start, retrying $count times"
```

```
Search mode master started
```

Reply:
thanks